handlers/error.go

```diff
@@ -61,6 +68,12 @@ var (
 	ErrValidAdapter     = errors.New(ErrValidAdapterCode, errors.Alert, []string{"Unable to find valid Adapter URL"}, []string{"unable to find a valid adapter for the given adapter URL"}, []string{"Given adapter URL is not valid"}, []string{"Please provide a valid Adapter URL"})
 	ErrAddAdapter       = errors.New(ErrAddAdapterCode, errors.Alert, []string{"meshLocationURL is empty"}, []string{"meshLocationURL is empty to add an adapter"}, []string{"meshLocationURL cannot be empty to add an adapter"}, []string{"please provide the meshLocationURL"})
 	ErrMeshClient       = errors.New(ErrMeshClientCode, errors.Alert, []string{"Error creating a mesh client", "Error pinging the mesh adapter"}, []string{"Unable to create a mesh client", "Unable to ping the mesh adapter"}, []string{"Adapter could not be pinged"}, []string{"Unable to connect to the Mesh adapter using the given config, please try again"})
+	ErrInvalidGenValue	= errors.New(ErrInvalidGenValueCode, errors.Alert, []string{"Invalid value for gen"}, []string{}, []string{}, []string{"please provide a valid value for gen (load generator)"})
+	ErrTestConfigs		= errors.New(ErrTestConfigsCode, errors.Alert, []string{"Error fetching test configs"}, []string{}, []string{}, []string{})
+	ErrReadDatabase		= errors.New(ErrReadDatabaseCode, errors.Alert, []string{"Error reading databases"}, []string{}, []string{}, []string{})
+	ErrWriteResponse	= errors.New(ErrWriteResponseCode, errors.Alert, []string{"Error writing response"}, []string{}, []string{}, []string{})
+	ErrField			= errors.New(ErrFieldCode, errors.Alert, []string{"field uuid not found"}, []string{}, []string{}, []string{})
+	ErrDeleteTestConfig	= errors.New(ErrDeleteTestConfigCode, errors.Alert, []string{"error deleting testConfig"}, []string{}, []string{}, []string{})
```

Comment:
already have a function called ErrFailToDelete() 
can use that instead creating one for each case